retroshare-gui/src/gui/settings/GroupFrameSettingsWidget.ui

```diff
@@ -40,6 +40,72 @@
      </property>
     </widget>
    </item>
+   <item>
+    <spacer name="verticalSpacer_2">

```

Comment:
Specify item name, don't use default one. How can we know reading text where is it?
And this help diff file block move.